zql/Makefile

```diff
@@ -9,9 +7,15 @@ run: all
 
 .PHONY: zql.go
 zql.go:
-	cpp -DGO -E -P zql.peg | $(PIGEON) -o $@
-	$(GOIMPORTS) -w $@
+	mkdir -p $(deps)
+	echo 'module deps' > $(deps)/go.mod
+	cd $(deps) && GOBIN=$(deps)/bin go get \
+		github.com/mna/pigeon@v1.0.1-0.20190520151103-9fec3898cef8 \
+		golang.org/x/tools/cmd/goimports@v0.0.0-20200204192400-7124308813f3
```

Comment:
Mostly curious, how were these particular versions chosen?

Reply:
They're the most recent versions available that would generate a zql.go identical to what's currently here.

Reply:
Eh, what's current here is really different from what we had been using until #265 was merged a few days ago which abruptly changed the version.

Reply:
Do you think we should switch to whatever was used before? (Chances are good that a change from pigeon v1.0.0 accounts for the difference.)

Reply:
The fact that 1.0.0 is a released version and the thing that we've done most of our testing with does feel better to me, but 🤷‍♂